### DISCUSSION TITLE
A Question about Using Multiple Apps with Flet

### DISCUSSION THREAD

**Discussion by sayak-roy-algo** [flet-dev/flet#4462] [Q&A]
Hello Flet community,

I am interested in using Flet to build realtime apps and I have a question. Is it possible to use multiple apps at a time without using multiprocessing? Additionally, is it possible to have common variables between screens, so that changing the value of a variable in one screen reflects on the other screen as well? I dont want to use shared objects using multiprocessing mangers or pipes or queues.

Thank you for your help!

**Comment by SKbarbon** [comment]
@sayak-roy-algo you like:
How to cook without using kitchen
